entrypoint/help_utils.go

```diff
@@ -100,7 +100,7 @@ func RegexpSplitAfter(re *regexp.Regexp, str string) []string {
 //        \tlogging into Gruntwork Houston
 func IndentAwareWrapText(text string, lineWidth int, indent string) string {
 	wrapped := ""
-	re := regexp.MustCompile("\\s+")
+	re := regexp.MustCompile(`\s+`)
 	words := RegexpSplitAfter(re, text)
```

Comment:
My editor was swearing all time. I had to change it. because it was annoying

```
In "help_utils_test.go":
should use raw string (`...`) with regexp.MustCompile to avoid having to escape twice [S1007]
In "help_utils.go":
should use raw string (`...`) with regexp.MustCompile to avoid having to escape twice [S1007]
In "help_utils.go":
should use raw string (`...`) with regexp.MustCompile to avoid having to escape twice [S1007]
In "help_utils.go":
should use raw string (`...`) with regexp.MustCompile to avoid having to escape twice [S1007]
```

Reply:
Ah, thanks for fixing these 👍